docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_9.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Incorporating GTEx Data in Kids First Analyses
+---
+
+<div class="banner"><span class="banner-text">Lesson in Development</span></div>
+
+Incorporating GTEx Data in Kids First Analyses
+==========================
+
+The Kids First DRC is a member of the [NIH Cloud Platform Interoperability Effort (NCPI)](https://anvilproject.org/ncpi), a collaboration of four NIH-funded DCCs to establish and implement guidelines and technical standards to empower end-user analyses across participating cloud platforms and facilitate the realization of a trans-NIH, federated data ecosystem. In brief, the goal of the NCPI Effort is to allow any authenticated user access to all of the platforms' datasets and the ability to work in the user's analysis platform of choice.
+
+As part of the NCPI Effort, methods have been developed to allow users to bring datasets from the other NIH platforms into CAVATICA for combined research projects. One external dataset that is supported is the [Genotype Tissue Expression (GTEx) Program](https://commonfund.nih.gov/gtex). GTEx was funded to study the relationship between genetic variants (inherited changes in DNA sequence) and gene expression (how genes are turned on and off) in multiple human tissues and across individuals. Their datasets can serve as great controls for RNA-Seq experiments, comparing expression in GTEx's "normal" brain tissue to Kids First's brain cancer tissue.
+
+The previous example provides a walkthrough for identifying RNA-Seq datasets from the [Kids First Data Resource Portal](https://portal.kidsfirstdrc.org/) and pushing them to [CAVATICA](https://cavatica.sbgenomics.com/) for analysis. This supplemental lesson will demonstrate how to find GTEx datasets on the [NHGRI Analysis Visualization and Informatics Lab-space (AnVIL)](https://anvilproject.org/) Portal] and import these files into a CAVATICA project for a combined analysis with Kids First data.
+
+
+## Step 1: Identify GTEx Files on AnVIL and Export a PFB
+- Navigate to [the AnVIL Portal](https://gen3.theanvil.io/login) and login using NIH or Google credentials.
+    - <img src="./rna-seq-images/rna-seq-9-01.png" width="400"/>
+- Using the [AnVIL Data Explorer](https://gen3.theanvil.io/explorer), identify the GTEx files for your analysis. For more information on using AnVIL Gen3, see [their support documentation here](https://anvilproject.org/learn/introduction/getting-started-with-gen3).
```

Comment:
On line 21, I would start the sentences with 'Click the "Explore Data" button to use the AnVIL Data Explorer....

Also, since this is a tutorial, I think you need to tell the reader exactly which GTEx files to identify. This way, they can follow along. At the end of the tutorial, you suggest that they go through it again with their files of interest

Reply:
added requested change for line 21, Click the "Explore Data" button to use AnVIL Data Explorer